docs/setup/settings.asciidoc

```diff
@@ -64,7 +64,7 @@ To enable SSL/TLS for outbound connections to {es}, use the `https` protocol
 in this setting.
 
 | `elasticsearch.logQueries:`
- | *deprecated* This setting is no longer used and will get removed in Kibana 8.0. Instead, set <<logging-verbose, `logging.verbose`>> to `true`
+ | *deprecated* This setting is no longer used and will get removed in Kibana 8.0.
```

Comment:
Should we document how to configure logging for `elasticsearch` service only?

Reply:
That's a good idea.

Reply:
I'll add an example. How about this:
```//kibana.yml
logging:
  appenders:
    console:
      type: console
      layout:
        type: pattern
        highlight: true
  root:
    appenders: [default, console]
    level: warn
  loggers:
    - name: elasticsearch.query
      level: debug
```